index.bs

```diff
@@ -414,23 +416,23 @@ The feature lists accept any string. However, in order to be a considered a vali
 
  - {{XRReferenceSpaceType}}
 
-Some {{XRSessionMode}}s enable certain [=feature names=] as [=optional features=] by default. This is only done if the feature does not require a signal of [=user intent=] nor impact performance or the behavior of other features when enabled. The following table describes the <dfn>default features</dfn> associated with each {{XRSessionMode}}:
+Some {{XRSessionMode}}s enable certain [=feature names=] as [=optional features=] by default. This is only done if the feature does not require a signal of [=user intent=] nor impact performance or the behavior of other features when enabled. The following table describes the <dfn>default features</dfn> associated with each session:
 
 <table class="tg">
   <thead>
     <tr>
       <th>Feature</th>
-      <th>{{XRSessionMode}}</th>
+      <th>Sessions</th>
     </tr>
   </thead>
   <tbody>
     <tr>
       <td>{{XRReferenceSpaceType/"viewer"}}</td>
-      <td>{{XRSessionMode/"inline"}}, {{XRSessionMode/"immersive-vr"}}, {{XRSessionMode/"immersive-ar"}}</td>
+      <td>[=Inline sessions=] and [=immersive sessions=]</td>
```

Comment:
I like this re-framing of which sessions the features apply to! It'll make the addition of `immersive-ar` in a separate module a lot cleaner. Thanks! :+1: